tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/TwilioAdapterTests.cs

```diff
@@ -65,13 +65,13 @@ public async void SendActivitiesAsyncShouldSucceedAndNoActivityReturnedWithActiv
             twilioApi.Setup(x => x.SendMessageAsync(It.IsAny<TwilioMessageOptions>(), default)).Returns(Task.FromResult(resourceIdentifier));
 
             var twilioAdapter = new TwilioAdapter(twilioApi.Object, _adapterOptions);
-            var resourceResponses = await twilioAdapter.SendActivitiesAsync(null, new Activity[] { activity.Object }, default).ConfigureAwait(false);
+            var resourceResponses = await twilioAdapter.SendActivitiesAsync(null, new Activity[] { activity.Object }, default);
```

Comment:
I hope removing the `.ConfigureAwait(false)` doesnt complain again at some point, maybe in the tests were not necessary? but in the libraries it always complain, i dont know if there is some rule.
Although, it looks better this way, less code.

Reply:
yes, following the doc of xnuit, this is not needed in the unit tests